temboardui/plugins/monitoring/model/tables.py

```diff
@@ -78,3 +79,16 @@
     Column('key', UnicodeText, nullable=True, primary_key=True),
     Column('state', UnicodeText, nullable=False),
     schema="monitoring")
+
+collector_status = Table(
+    'collector_status',
+    metadata,
+    Column('instance_id', Integer,
+           ForeignKey("monitoring.instances.instance_id"),
+           primary_key=True),
+    Column('last_pull', DateTime, nullable=True),
+    Column('last_push', DateTime, nullable=True),
+    Column('last_insert', DateTime, nullable=True),
```

Comment:
We need both because both modes are still available like PR description says. BTW, having both will help for troubleshooting to known which mode is active for each agent.

Reply:
Ok, makes sense. Thanks for the explanation.

I reread the full diff and now see that `last_pull` is actually used. (I missed it the first time because large diffs were hidden by github's ui.)